src/NetworkManager.cpp

```diff
@@ -0,0 +1,63 @@
+#include "NetworkManager.h"
+#include "util/config.h"
+
+#include <GLFW/glfw3.h>
+
+NetworkManager::NetworkManager() {
+    Config::config->get_value(Config::str_is_server, is_server);
+    Config::config->get_value(Config::str_server_ip, server_host);
+    Config::config->get_value(Config::str_port, port);
+
+    server = is_server ? new NetworkServer(&io_service, port) : nullptr;
```

Comment:
I'd like to have new since I have server and client objects as class variables, so their initial values are null pointers.